receiver/azureeventhubreceiver/README.md

```diff
@@ -95,10 +95,11 @@ and the OpenTelemetry attributes.
 | resultSignature (optional)       | azure.result.signature (attribute)     | 
 | resultType (optional)            | azure.result.type (attribute)          | 
 | tenantId (required, tenant logs) | azure.tenant.id (attribute)            | 
-| time (required)                  | time_unix_nano (field)                 | 
+| time or timeStamp (required)     | time_unix_nano (time takes precedence) |
```

Comment:
It would make more sense to me to have the `time takes precence comment` in the same column as `time or timeStamp`, but this is just personal preference, this should be generally understandable.